src/test/ui/consts/control-flow/drop-fail.rs

```diff
@@ -1,11 +1,14 @@
+// revisions: stock precise
+
 #![feature(const_if_match)]
 #![feature(const_loop)]
+#![cfg_attr(precise, feature(const_precise_live_drops))]
 
-// `x` is *not* always moved into the final value may be dropped inside the initializer.
+// `x` is *not* always moved into the final value and may be dropped inside the initializer.
 const _: Option<Vec<i32>> = {
     let y: Option<Vec<i32>> = None;
```

Comment:
If we unconditionally error on drops, this line will cause an error (meaning: this is the test case you're looking for)

Reply:
If I interpret the error annotations correctly, this errors (in the next line) even with `precise` analysis? Doesn't seem terribly useful then to not error here.^^

Isn't there an example where we need this to actually accept more code?

Reply:
The next line is completely independent of this line.

Reply:
I think I'll split up the feature gate into two, so we can actually play with examples on the playground.

Reply:
I don't think we want this exposed to the user; doesn't just commenting out those lines make some tests fail (in more ways than just moving around the location of the error message)?

Reply:
I found examples in the test suite: https://github.com/rust-lang/rust/pull/83351